common/map_file/nodes/lanelet2_map_loader/lanelet2_map_loader.cpp

```diff
@@ -63,18 +69,18 @@ int main(int argc, char** argv)
 
   lanelet::ErrorMessages errors;
   lanelet::LaneletMapPtr map;
-
+  
   int projector_type = 0;
-  private_nh.param<int>("projector_type", projector_type, 1);
+  std::string base_frame , target_frame;
+  // Parse geo reference info from the lanelet map (.osm)
+  lanelet::io_handlers::AutowareOsmParser::parseMapParams(lanelet2_filename, &projector_type, &base_frame, &target_frame);
```

Comment:
They are passed by reference in the parseMapParams to get two different frames out of the function.

Reply:
Projector type does not seem to be used though. I mainly kept it because it was on original yaml config file, and it is not even broadcasted I believe. Please let me know if you think it is useful or not.

Reply:
I was more asking, why they need to be set here, but it looks like this is just to make this call conform to your new function specification.